web/src/server/api/routers/auditLogs.ts

```diff
@@ -83,11 +83,13 @@ export const auditLogsRouter = createTRPCRouter({
             image: true,
           },
         }),
+        // TODO: Do we also need to join organization keys here or indicate the scope on the result?
```

Comment:
Comment to highlight todo -> This mainly controls how things are displayed in the audit log table in the frontend.